packages/tsutils/src/period/period.ts

```diff
@@ -19,3 +34,111 @@ export const PERIOD_TYPES = {
   QUARTER, // e.g. '2018Q1'
   YEAR, // e.g. '2018'
 } as const;
+
+const NON_NUMERIC_PERIOD_TYPES = [WEEK, QUARTER];
```

Comment:
Everything here (except for types) is copied over from `period.js` in `@tupaia/utils`